src/Servers/Kestrel/test/InMemory.FunctionalTests/CertificateLoaderTests.cs

```diff
@@ -12,6 +14,9 @@ namespace Microsoft.AspNetCore.Server.Kestrel.InMemory.FunctionalTests
 {
     public class CertificateLoaderTests : LoggedTest
     {
+        //private const string ServerAuthenticationEnhancedKeyUsageOid = "1.3.6.1.5.5.7.3.1";
```

Comment:
Why are the changes to this file commented out?

Reply:
CI don't allow saving certificates, so all newly added tests were commented out, this line relates to these tests. 

> security: SecKeychainItemImport: User interaction is not allowed.

We can remove all commented lines from it, left them just to allow run locally at least.

Reply:
@Tratcher how would we normally do this kind of automated testing?

Reply:
Please remove these commented out parts.

Some code that interacts with shared resources can't practically be unit tested. There are cases where we'll check in a manual test and mark it as Skipped or quarantined. However, these tests directly modify machine state and that's not something we should check in even as a manual test.

If we were worried about regressing this then we'd document some manual test procedures and get someone to run them periodically. This is enough of a corner case that I don't expect it to change much, nor do I expect it to be worth the cost of ongoing manual verification.

Reply:
Sure, I'll revert it to original state